docs/02-get-started/01-quick-install.md

```diff
@@ -38,26 +38,14 @@ export CLUSTER_DOMAIN={YOUR_CLUSTER_DOMAIN}
 
 > **NOTE:** For local installation, the cluster domain is `local.kyma.dev`.
 
-## Connect to Kyma Dashboard
+## Open Kyma Dashboard
 
-To manage Kyma via GUI, connect it to Kyma Dashboard. 
+To manage Kyma via GUI, open Kyma Dashboard:
 
-1. To start the Dashboard, run:
-
-    ```bash
+```bash
     kyma dashboard
```

Comment:
Please fix the indentation here as well. In the preview, you can see that it is rendered indented, and it shouldn't be. 
![image](https://user-images.githubusercontent.com/53601578/142870419-d1c3fbf2-d9c5-43c7-9fbb-a2676414eb0c.png)
```suggestion
kyma dashboard
```